packages/@react-stately/dnd/src/useDraggableCollectionState.ts

```diff
@@ -48,50 +59,54 @@ export function useDraggableCollectionState(props: DraggableCollectionOptions):
   };
 
   return {
-    collection: props.collection,
-    selectionManager: props.selectionManager,
+    collection,
+    selectionManager,
     isDragging(key) {
       return draggingKeys.has(key);
     },
     getKeysForDrag: getKeys,
     getItems(key) {
-      return props.getItems(getKeys(key));
+      if (typeof getItems === 'function') {
```

Comment:
when is getItems not a function? maybe a code comment to explain this case?

Reply:
`getItems` is an optional prop so it can be undefined and the above was a more specific guard against that case. @reidbarber Perhaps we should move the default for `getItems` defined in useDragHooks to this hook instead? What do you think?

Reply:
@LFDanLu I think that makes more since. And the default for `itemAllowsDragging` too.